### PR TITLE
Improve coding style

### PR DESCRIPTION
**Pull request by windowsair**
Hi,

Thank you for your excellent work. I really like dapjs, but it looks like it hasn't been updated for a while, and now there are some dependencies that haven't been updated for a long time, and some dependencies that have been deprecated.

I'm currently trying to update the dependencies to the latest, but I'm not sure if this repo still accepting contributions from the community, so let's start with the coding style :D

### CODE REVIEW

File: src/transport/usb.ts [hunk 1]
```diff
@@ -195,7 +195,11 @@ export class USB implements Transport {
                     if (exception) {
                         reject(exception);
                     } else {
-                        resolve(buffer);
+                        if (buffer !== undefined) {
```
Comment (thegecko) [inline src/transport/usb.ts:198]: This is a coding change, perhaps pop it in a separate PR?

File: src/transport/usb.ts [hunk 2]
```diff
@@ -195,7 +195,11 @@ export class USB implements Transport {
...
+                            resolve(buffer);
```
Comment (thegecko) [inline src/transport/usb.ts:199]:
```suggestion
                            resolve(buffer as Buffer);
```

 ? :)

**Review by thegecko — CHANGES_REQUESTED** [review]
Thanks for your PR, just a minor suggestion and recommendation to split the work.

**Comment by windowsair** [comment]
Thank you for your fast review! There are indeed some changes in usb.ts that tend to be confusing, that are basically changes to adapt the new version, so for now I'll remove them. Then, according to your guidance, I have put some code -related modifications to the patch 2.

File: src/transport/usb.ts (hunk 1 above)
Comment (windowsair) [reply src/transport/usb.ts:198]: Done

File: src/transport/usb.ts (hunk 2 above)
Comment (windowsair) [reply src/transport/usb.ts:199]: Removed this

**Comment by thegecko** [comment]
> ...according to your guidance, I have put some code -related modifications to the patch 2.

Thanks, can you pout it in a separate PR, so this PR is only code style changes?

**Comment by windowsair** [comment]
> > ...according to your guidance, I have put some code -related modifications to the patch 2.
> 
> Thanks, can you pout it in a separate PR, so this PR is only code style changes?

Sure, I will push them later. Now this PR only has coding style commit.

**Review by thegecko — APPROVED** [review]

**Pull request merged**